rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -25,6 +25,7 @@ spec:
             description: >-
               Tekton controller on cluster {{ $labels.source_cluster }} the percentage of time needed to receive PipelineRun creation
               events vs. overall PipelineRun execution time is at {{ $value | humanizePercentage }} instead of less than 5%.
+            alert_route_namespace: plnsvc-tests
```

Comment:
```suggestion
            alert_route_namespace: pipeline-service
```
For some consistency. Also, why include the `-tests` postfix?

Reply:
That's an actual namespace name. I'm not aware of `pipeline-service` namespace.

Reply:
as a result, will not fix. 
is fixed later on #185